technologies/job/gcp-cloud-functions/project/context.yaml

```diff
@@ -0,0 +1,47 @@
+id: project
+label: project
+description: ""
+recommended: true
+trustLevel: experimental
+endpoint:
+  features:
+    - type: TEXT
+      name: jsonKey
+      label: JSON key
+      required: true
+      helper: "Full JSON document"
+    - type: SELECT
+      name: location
+      label: Location
+      required: true
+      options:
+        script: ./endpointForm.js
+        function: getLocations
+      dependsOn:
+        - jsonKey
```

Comment:
This filter should be in the job form part.

May be optional for not so much functions contexts.

Reply:
+ we suggest "Region" instead of location to respect Google wording.